config/application.rb

```diff
@@ -17,6 +17,7 @@ class Application < Rails::Application
     config.i18n.fallbacks = true
 
     config.middleware.use "Redirector" unless Rails.env.development?
+    config.middleware.use Rack::Attack
```

Comment:
I think this make sense only in production (or at least it shouldn't be present in test env).